src/mono/wasm/runtime-test.js

```diff
@@ -127,8 +127,8 @@ while (true) {
 		var arg = args [0].substring ("--runtime-arg=".length);
 		runtime_args.push (arg);
 		args = args.slice (1);
-	} else if (args [0] == "--enable-gc") {
-		enable_gc = true;
+	} else if (args [0] == "--disable-gc") {
```

Comment:
I wonder if we should rename this to `--disable-ondemand-gc`, cause it doesn't really "disable" the GC completely like the name would suggest.